docs(failures): update and create new pages (#862)
flybywiresim/docs: docs/fbw-a32nx/feature-guides/flypados3/failures.md

It may be configured either to trigger while the plane accelerates or decelerates.

![failures-speed.png](../../assets/flypados3/failures-speed.png){loading=lazy}

### Timed Failure Generator

This generator triggers a set of failures among the selected failure pool when the time since the arming of the generator reaches a random delay set between the specified minimum and maximum delay.

- When configured in the "repeat" or "Once" arming modes, the timer will start right away.
- When configured in the "Take Off" arming, the timer will start once FLEX or TOGA thrust is set for the first time.

![failures-timed.png](../../assets/flypados3/failures-timed.png){loading=lazy}

### Probability Over Time Generator

At any moment, this generator may trigger a set of failures among the selected failure pool by using the probability specified through the average number of failure per hour.
The mean time to failure (MTTF) is provided as an indication of the delay it may take in average to trigger one failure for the probability of failure per hour.

!!! note ""
    Rates of failure in aviation vary between 10e-3 per hour and 10e-9 per hour depending on the criticality of the system.

> For simulation purposes, it is advised to set this value at a much higher level up to 10e-1 (0.1 failure / hour or 1 failure every 10 hours) in order to experience failures once every simulator session.

Reviewer: ```suggestion
For simulation purposes, it is advised to set this value at a much higher level, up to 10e-1 (0.1 failure / hour or 1 failure every 10 hours) in order to experience failures once every simulator session.
```

![failures-all-systems.png](../../assets/flypados3/failures-all-systems.png){loading=lazy}

### Take Off Failure Generator

This generator triggers a set of failures among the selected failure pool with a specified probability at each take off.

> If a failure will occur during the next take off, it may happen at any moment, during one of these three take off phases:

Reviewer: ```suggestion
If a failure occurs during the next take-off, it may happen at any moment, during one of these three take off phases:
```

- Low ground speed phase :

>     - The take off phase when the plane exceeds the minimum ground speed and is slower than the low-med transition ground speed.
> - Medium ground speed phase :
>     - The take off phase when the plane exceeds the low-med transition ground speed and is slower than the med-high transition ground speed.
> - High ground speed & climb phase :
>     - The take off phase when the plane exceeds the med-high transition ground speed and is lower than the specified height above the runway. 

Reviewer: ```suggestion
    - The take-off phase when the plane exceeds the minimum ground speed and is slower than the low-med transition ground speed.
- Medium ground speed phase :
    - The take-off phase when the plane exceeds the low-med transition ground speed and is slower than the med-high transition ground speed.
- High ground speed & climb phase :
    - The take-off phase when the plane exceeds the med-high transition ground speed and is lower than the specified height above the runway. 
```

    <a href="../online-atc/">  <div class="imagemap" style="position: absolute; left: 1.7%; top: 43.0%; width: 5.8%; height: 7.0%;"><span class="imagemapname">Online ATC</span></div></a>
    <a href="failures">    <div class="imagemap" style="position: absolute; left: 1.7%; top: 50.1%; width: 5.8%; height: 7.0%;"><span class="imagemapname">Failures</span></div></a>
    <a href="../checklists/">  <div class="imagemap" style="position: absolute; left: 1.7%; top: 57.3%; width: 5.8%; height: 7.0%;"><span class="imagemapname">Checklists</span></div></a>
    <a href="../presets/">     <div class="imagemap" style="position: absolute; left: 1.7%; top: 64.7%; width: 5.8%; height: 7.0%;"><span class="imagemapname">Presets</span></div></a>
    <a href="../settings/">    <div class="imagemap" style="position: absolute; left: 1.7%; top: 85.0%; width: 5.8%; height: 7.0%;"><span class="imagemapname">Settings</span></div></a>
    <span class="imagesub">Click on the menu icons in this image to see other flyPad pages.</span>
</div>

## Failure Generator

> These optionss are found on the "Failure Generators" tab on the Failure page.

Reviewer: ```suggestion
These options are found on the "Failure Generators" tab on the Failure page.
```

!!! info ""

>     Please ensure that you are on the correct tab to trigger set up the correct generator.

Reviewer: Either trigger or set up

>     Please ensure that you are on the correct tab to trigger set up the correct generator.

Author: ```suggestion
    Please ensure that you are on the correct tab to trigger or set up the correct generator.
```

    ![flypad-failure-tab.png](../../assets/flypados3/flypad-failure-tab.png){loading=lazy}

-- TODO -- 

- Add short overview here
- add images of each page on the failures page

!!! note "Configuration Options"
    Each generator can be configured in the following 4 modes:

    - OFF : Generator is disabled but settings are kept
    - ONCE : The generator will trigger a set of failures only once and go to OFF mode
    - TAKE OFF : The generator will wait until FL or TOGA thrust is set to arm itself. It will trigger a set of failures only once until another take off event occurs.
    - REPEAT : The generator will trigger a set of failures every time the conditions are met.

### Max Failures and Number of Failures

Each generator defines a specific number of failures to be trigger at the same time and a maximum number of failures.

> The number of failures will activate up to the total number of failures configured for the flight.

Reviewer: ```suggestion
Failures will activate up to the total number of failures configured for the flight.
```

### Failure Pool Selection

Each failure generator allows to select specifically which failures can be triggered. You may select any combination of failures: 

- Individual failures
- Entire systems 
- or all at once. 
 

> Only a specified number of failures randomly picked in this set will be triggered by the generator.

Reviewer: ```suggestion
Only a specified number of failures, randomly picked from this set, will be triggered by the generator.
```

!!! tip ""
    You can create several failure generators of the same type so that specific failures may happen at specific conditions different to other sets of failures.

### Altitude Failure Generator

This generator triggers a set of failures among the selected failure pool when the airplane reaches a random altitude set between the specified minimum and maximum altitude.

It may be configured either to trigger while the plane climbs or descends.

### Ground Speed Failure Generator

This generator triggers a set of failures among the selected failure pool when the airplane reaches a random ground speed set between the specified minimum and maximum ground speed.

It may be configured either to trigger while the plane accelerates or decelerates.

### Timed Failure Generator

This generator triggers a set of failures among the selected failure pool when the time since the arming of the generator reaches a random delay set between the specified minimum and maximum delay.

- When configured in the "repeat" or "Once" arming modes, the timer will start right away.

> - When configured in the "Take Off" arming, the timer will start once FL or TOGA thrust is set for the first time.

Reviewer: ```suggestion
- When configured in the "Take Off" arming, the timer will start once FLEX or TOGA thrust is set for the first time.
```
